ESMF/GOCART2G_GridComp/DU2G_GridComp/DU2G_StateSpecs.rc

```diff
@@ -2,84 +2,104 @@ schema_version: 2.0.0
- LWI        | 1      | xy   | N   |      | land-ocean-ice_mask
- FRLAKE     | 1      | xy   | N   |      | fraction_of_lake
- U10M       | m s-1  | xy   | N   |      | 10-meter_eastward_wind
- V10M       | m s-1  | xy   | N   |      | 10-meter_northward_wind
- AREA       | m^2    | xy   | N   |      | agrid_cell_area
- USTAR      | m s-1  | xy   | N   |      | equivalent_neutral_10-meter_northward_wind
- CN_PRCP    |kg/m^2/s| xy   | N   |      | surface_conv._rain_flux_needed_by_land
- NCN_PRCP   |kg/m^2/s| xy   | N   |      | Non-convective precipitation
- ZPBL       | m      | xy   | N   |      | planetary_boundary_layer_height
- SH         | w m-2  | xy   | N   |      | sensible_heat_flux_from_turbulence
- Z0H        | m      | xy   | N   |      | surface_roughness_for_heat
+#------------------------------------------------------------------------------------------
+#  VARIABLE            | DIMENSIONS  |          Additional Metadata
+#------------------------------------------------------------------------------------------
+     NAME   | UNITS    | DIMS | VLOC | COND                                     | LONG NAME
+#------------------------------------------------------------------------------------------
+ DU_SRC     | 1        | xy   | N    | scheme == 'ginoux' .OR. scheme == 'k14'  | erod - dust emissions
+ DU_Z0      | 1        | xy   | N    | scheme == 'k14'                          | aerodynamic_surface_roughness_for_aeolian_processes
+ DU_GVF     | 1        | xy   | N    | scheme == 'k14'                          | GVF
+ DU_SAND    | 1        | xy   | N    | scheme == 'fengsha' .OR. scheme == 'k14' | volume_fraction_of_sand_in_soil
+ DU_SILT    | 1        | xy   | N    | scheme == 'fengsha' .OR. scheme == 'k14' | volume_fraction_of_silt_in_soil
+ DU_CLAY    | 1        | xy   | N    | scheme == 'fengsha' .OR. scheme == 'k14' | volume_fraction_of_clay_in_soil
+ DU_RDRAG   | m-1      | xy   | N    | scheme == 'fengsha'                      | drag_partition
+ DU_SSM     | 1        | xy   | N    | scheme == 'fengsha'                      | sediment_supply_map
+ DU_UTHRES  | m s-1    | xy   | N    | scheme == 'fengsha'                      | surface_dry_threshold_velocity
+ FRSNOW     | 1        | xy   | N    | scheme == 'fengsha'                      | surface_snow_area_fraction
+ SLC        | 1        | xy   | N    | scheme == 'fengsha'                      | liquid_water_content_of_soil_layer
+ DU_TEXTURE | 1        | xy   | N    | scheme == 'k14'                          | soil_texture
+ DU_VEG     | 1        | xy   | N    | scheme == 'k14'                          | vegetation_type
+ FRLAKE     | 1        | xy   | N    |                                          | fraction_of_lake
+ FRLAND     | 1        | xy   | N    |                                          | fraction_of_land
+ ASNOW      | 1        | xy   | N    |                                          | snow_covered_fraction_of_land
```

Comment:
`ASNOW` and `FRLAND` are only used by the k14 dust scheme. Please set as:
```
FRLAND     | 1        | xy   | N    | scheme == 'k14'                          | fraction_of_land
ASNOW      | 1        | xy   | N    | scheme == 'k14'                          | snow_covered_fraction_of_land
```

Reply:
Please also set:
```
 TSOIL1     | K        | xy   | N    | scheme == 'k14'                          | soil_temperatures_layer_1
 RHOS       | kg m-3   | xy   | N    | scheme == 'k14'                          | air_density_at_surface
```

Reply:
@rmontuoro Done and tested as zero-diff to ginoux and k14. Please re-test with fengsha